buildscripts/incremental/setup_conda_environment.sh

```diff
@@ -63,9 +63,12 @@ if [ "${VANILLA_INSTALL}" != "yes" ]; then
 fi
 
 # Install the compiler toolchain
+arch=`uname -m`
 if [[ $(uname) == Linux ]]; then
     if [[ "$CONDA_SUBDIR" == "linux-32" || "$BITS32" == "yes" ]] ; then
         $CONDA_INSTALL gcc_linux-32 gxx_linux-32
+    elif [[ "$arch" == "ppc64le" ]] ; then
+        $CONDA_INSTALL gcc_linux-"$arch" gxx_linux-"$arch"
```

Comment:
```suggestion
        # This branch is not exercised by Numba's Azure CI for the main fork.
        $CONDA_INSTALL gcc_linux-"$arch" gxx_linux-"$arch"
```

Reply:
Don't understand this, to be honest, but pressed the button anyways.